### ISSUE TITLE
CC1101 isPromiscuous is broken

### ISSUE THREAD

**Issue by bazuchan** [rfquack/RFQuack#36]
isPromiscuous implementation sets PreambleLen to 0 here
https://github.com/rfquack/RadioLib/blob/master/src/modules/CC1101/CC1101.cpp#L828

which is considered as invalid value
https://github.com/rfquack/RadioLib/blob/master/src/modules/CC1101/CC1101.cpp#L687

**Comment by phretor** [comment]
Good catch. I'm afraid this will have to be addressed with a special case.
